Support/Scripts/install-android-emulator.sh

```diff
@@ -9,17 +9,29 @@
 ## PATENTS file in the same directory.
 ##
 
-set -eu
+source "$(dirname "$0")/common.sh"
 
-android_script="/tmp/android-sdk-linux/tools/android"
+[ $# -eq 0 ] || die "usage: $0"
+
+case "$(uname)" in
+  "Linux")  platform_name="linux"; package_extension="tgz";;
+  "Darwin") platform_name="macosx"; package_extension="zip";;
+  *)        die "This script works only on Linux and macOS.";;
+esac
+
+android_sdk_version="r24.4.1"
+android_script="/tmp/android-sdk-${platform_name}/tools/android"
 
 if [ ! -f "$android_script" ] ; then
-  android_sdk_name="android-sdk_r24.4.1-linux"
-  wget "https://dl.google.com/android/$android_sdk_name.tgz" -P "/tmp/"
-  tar -zxf "/tmp/$android_sdk_name.tgz" -C "/tmp/"
+  android_sdk_package="android-sdk_${android_sdk_version}-${platform_name}.${package_extension}"
+  wget "https://dl.google.com/android/${android_sdk_package}" -P /tmp
+  case "${package_extension}" in
+    "zip") unzip -d /tmp "/tmp/${android_sdk_package}";;
+    *)     tar -C /tmp -xvf "/tmp/${android_sdk_package}";;
```

Comment:
I'd make this an explicit "tgz", in case we add other platforms, etc in the future